docker-compose.yml

```diff
@@ -6,8 +6,27 @@ services:
       - 8080:80
     volumes:
       - ./nginxconf/default.conf:/etc/nginx/conf.d/default.conf
-      - ./nginxconf/nginx.conf:/usr/local/openresty/nginx/conf/nginx.conf
+      - ./nginxconf/nginx.conf:/usr/local/openresty/nginx/conf/nginx.conf    
+    networks:
+      frontend:
+        ipv4_address: 172.20.0.2
   au:
     image: otomato/au
     ports: 
       - 3100
+    networks:
+      frontend:
+        ipv4_address: 172.20.0.3
+  syslog:
+    image: kuzishai/syslog-server-udp
```

Comment:
Please use a specific tag (e.g 0.1) for docker images you build.
I know I didn't do this for otomato/au :)) Shoemaker goes barefoot.
I will fix this on my side.

Reply:
done[commit](https://github.com/mondbev1/nginx-test-rig/commit/5c7aa0af60e0a1dfeaf2f3234ae697aa2f78218c)